recipe/0001-inject-cmake-project.patch

```diff
@@ -64,6 +64,9 @@ index 0000000..c15df47
 +                            $<INSTALL_INTERFACE:include>
 +                           )
 +
++target_compile_definitions(getopt PRIVATE EXPORTS_GETOPT _CONSOLE)
++target_compile_options(getopt PRIVATE /Oi /O2 /W3 /Gy)
```

Comment:
Why do you need /W3?

Why not let CMake control /O2 with the build type?

Reply:
Another emulation of the flags in https://github.com/libimobiledevice-win32/getopt/blob/master/getopt.vcxproj#L88 in both cases https://github.com/libimobiledevice-win32/getopt/blob/master/getopt.vcxproj#L81

Reply:
done